docs/source/models.rst

```diff
@@ -1,10 +1,10 @@
 .. _models:
 
-torchvision.models
-##################
+Models and pre-trained weights
```

Comment:
Probably write in Title Case. As that's how PyTorch docs on left are

![image](https://user-images.githubusercontent.com/47158509/139534539-c525c776-714d-4b70-9843-ef3a35c49d3d.png)

Reply:
I'm not sure they're using Title Case - see e.g. "Features or large-scale deployments" or "Autograd mechanics".

They're suing "Automatic Mixed Prececision" and "Distributed Data Parallel" and "Frequently Asked Questions" because these are commonly known and shortened AMP,  DDP, FAQ, etc.